security/netbird/src/opnsense/mvc/app/models/OPNsense/Netbird/Status.xml

```diff
@@ -0,0 +1,6 @@
+<model>
+    <mount>//OPNsense/netbird/status</mount>
```

Comment:
why do we need a persistent model here if it's empty? in some cases `:memory:` makes sense but for now this doesn't even have items

Reply:
```suggestion
    <mount>:memory:</mount>
```

this should do the trick and not render in config.xml until we know what to do